File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/FavoriteCheckVO.java
package com.sdkwork.app.model;

public class FavoriteCheckVO {
    private String createdAt;
    private String updatedAt;
    private String targetId;
    private Boolean isFavorite;
    private String favoriteId;
    private String favoriteTime;

    public String getCreatedAt() {
        return this.createdAt;
    }
    
    public void setCreatedAt(String createdAt) {
        this.createdAt = createdAt;
    }

    public String getUpdatedAt() {
        return this.updatedAt;
    }
    
    public void setUpdatedAt(String updatedAt) {
        this.updatedAt = updatedAt;
    }

    public String getTargetId() {
        return this.targetId;
    }
    
    public void setTargetId(String targetId) {
        this.targetId = targetId;
    }

    public Boolean getIsFavorite() {
        return this.isFavorite;
    }
    
    public void setIsFavorite(Boolean isFavorite) {
        this.isFavorite = isFavorite;
    }

    public String getFavoriteId() {
        return this.favoriteId;
    }
    
    public void setFavoriteId(String favoriteId) {
        this.favoriteId = favoriteId;
    }

    public String getFavoriteTime() {
        return this.favoriteTime;
    }
    
    public void setFavoriteTime(String favoriteTime) {
        this.favoriteTime = favoriteTime;
    }
}

File: sdkwork-app-sdk-typescript/src/types/favorite-check-vo.ts
/** 收藏检查响应 */
export interface FavoriteCheckVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 收藏对象ID */
  targetId?: string;
  /** 是否已收藏 */
  isFavorite?: boolean;
  /** 收藏ID */
  favoriteId?: string;
  /** 收藏时间 */
  favoriteTime?: string;
}
